### DISCUSSION TITLE
Granite Artefact scan data and electrical grounding

### DISCUSSION THREAD

**Discussion by barkingbear349** [markqvist/Reticulum#304] [Ideas]
This is about the vase and the scan data.

If you google the numbers of the radial traversal function you see multiple correlations with the following chart on this site:  https://www.qsl.net/ac5jw/isbp/vf-gen.html

It is from the following project.
The Integrated Systems Bonding Project is an informal treatise on electrical grounding and safety

Has any one more to say about this, are these correlating numbers just because i used the speed of light and not 18.74 mm ?

**Comment by markqvist** [comment]
You found that page because it has a huge table of many different RF wavelengths, one of them being the 18.737mm wavelength of a 16Ghz EM-wave propagating in vacuum.

It's got a lot of useful stuff for calculating correct grounding setups for all kinds of RF systems.

**Reply by AppalachianWarrior** [reply]
Hi Mark. Thanks for sharing your insights and the reference page with the RF wavelength data.

I read your vase report at https://unsigned.io/granite-artefact/

I wanted to share a perspective regarding the frequency (around 18.7 mm wavelength) associated with the granite vase. While it’s true that 18.7 mm corresponds to about 16 GHz in electromagnetic waves propagating in vacuum, I think this artifact’s context suggests a different interpretation.

Since the vase is a solid object with a hollow cavity, and if it functioned as a sound related device and not an EM/RF related device, the relevant frequency should correspond to **sound waves** traveling through the medium inside it—most likely air. The speed of sound in air (~343 m/s) leads to a frequency around 18 kHz for an 18.7 mm wavelength, which is well within the range of human hearing and ultrasonic sound. ( Acoustic resonance depends on the cavity geometry. The scan shows complex internal **contours**—which apparently define harmonic frequencies. )

Also, the handles' positions derive from 1 radian arcs. Radians directly relate to phase in wave mechanics. I think this piece of info should not be overlooked.  

On the other hand, applying EM/RF principles—using the speed of light (~3×10^8 m/s)—to this physical object is less direct, as the vase itself is granite, not a conductor or typical RF structure.

Given:
- Wavelength (λ) = 18.7391 mm = 0.0187391 meters
- Speed of sound in air (v) ≈ 343 m/s (at room temperature)

Formula:
f = v / λ
---
f = 343 m/s ÷ 0.0187391 m ≈ 18,302 Hz
---
Result:
- The resonant frequency inside the vase is approximately 18.3 kHz.
- This frequency is in the range of human hearing.

- This suggests the vase’s geometry may have been intentionally designed to resonate at human audible frequencies.

_( I do know that the resonance frequency of a hollow object like this vase depends on the shape, volume, and dimensions of the internal cavity and not just the size of the opening. But I am trying to keep things simple to advocate that this is a sound based, rather than an RF/EM system )_

So, the math and physical context together suggest that the vase’s design might be linked to acoustic resonance rather than electromagnetic or RF phenomena. This also aligns better with known applications of sound like sonic tools. Also, carving this granite vase to µm precision is only justified if the maker was targeting resonant sound frequencies , where tiny errors would shift the tones.

I hope this clarifies my thinking, and I appreciate your openness to discussing different interpretations. I’d love to hear your thoughts on this as well!
